languages/rust/reference/exercise-concepts/paasio.md

```diff
@@ -0,0 +1,187 @@
+# Concepts of Paasio
+
+PaaS I/O explores extending standard `Read` and `Write` traits to provide additional statistics via a wrapper type. A good way to learn about traits in Rust and their best features.
+
+This exercise can be implemented using either a single or two (for `Read` and `Write`)
+newtype structs, with appropriate trait bounds.
```

Comment:
```suggestion
types, with appropriate trait bounds.
```

Neither example actually uses the [newtype pattern](https://doc.rust-lang.org/1.0.0/style/features/types/newtype.html).